packages/i18n/next-i18next.config.js

```diff
@@ -4,7 +4,7 @@ const path = require('path')
 const config = {
   i18n: {
     // Add your language ISO 639-1 code here.
-    locales: ['en', 'it'],
+    locales: ['en', 'it', 'pl', 'uk', 'zh'],
```

Comment:
Can we add just the `zh` to this PR?

(I'm going to merge this first and get a second Mandarin-speaking reviewer to give it a pass).

Reply:
IMO, let's get these all in and then try to get feedback from those who speak the language.